components/conversations/CommentForm.js

```diff
@@ -83,18 +105,25 @@ const CommentForm = ({
   LoggedInUser,
   isDisabled,
 }) => {
-  const [createComment, { error, data }] = useMutation(createCommentMutation, mutationOptions);
-  const { register, triggerValidation, setValue, formState, handleSubmit } = useForm({ mode: 'onChange' });
+  const [createComment, { mutationError, data }] = useMutation(createCommentMutation, mutationOptions);
   const { formatMessage } = useIntl();
+  const [state, dispatch] = useReducer(reducer, initialFormState);
```

Comment:
**[:sparkles: **Enhancement**]** There is just one field, `html`. Creating a reducer for the form state seems overkill to me, it defeats the simplicity of hooks by re-creating a `setState` with a big object containing all the values. Why not just use `useState`?

```suggestion
  const [html, setHtml] = React.useState('');
  const [validationError, setValidationError] = React.useState(null);
```

---

<details>
<summary>More details about this feedback</summary>
<br/>

This feedback categorization is inspired by [Netlify's Feedback Ladders system](https://www.netlify.com/blog/2020/03/05/feedback-ladders-how-we-encode-code-reviews-at-netlify). It includes 5 levels of feedback:

:rotating_light: **Critical blocker**
This feedback blocks all related work, and requires immediate action. 

:no_entry_sign: **Blocker**
This feedback should block the work from moving forward or being approved, but it does not necessarily require immediate action from the team.

:hammer_and_wrench: **Important**
This feedback should not block the work from moving forward or being approved, but it does require future action from the team. This generally means the feedback is not required for the MVP or initial draft of this work, but should be addressed at some point and we should ideally create an issue to track it.

:sparkles: **Enhancement**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they agree with it.

:ice_cream: **Details & personal preferences**
This feedback should not block the work from moving forward or being approved. The project lead or owner should consider the feedback and implement or open an issue it if they deem it useful. This is also the category for everything that relates to personal preferences or code style.

</details>

Reply:
You're right it is kind of overkill. I had been trying to familiarise myself with hooks since I haven't implemented them much before, looks like I got carried away in the process.